x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/monitor_add_edit/hooks/use_clone_monitor.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useFetcher } from '@kbn/observability-shared-plugin/public';
+import { useGetUrlParams } from '../../../hooks';
+import { getDecryptedMonitorAPI } from '../../../state/monitor_management/api';
+
+export const useCloneMonitor = () => {
+  const { cloneId } = useGetUrlParams();
+  const { data, loading, error } = useFetcher(() => {
+    if (!cloneId) return Promise.resolve(undefined);
+    return getDecryptedMonitorAPI({ id: cloneId });
+  }, [cloneId]);
+  return { data, loading, error };
```

Comment:
```suggestion
  return useFetcher(() => {
    if (!cloneId) return Promise.resolve(undefined);
    return getDecryptedMonitorAPI({ id: cloneId });
  }, [cloneId]);
```

Why destructure this object and then return the exact same object? This is potentially destructuring `undefined` in the case where the `cloneId` is falsy, I can't remember if will break, but it's not ideal and should use optional chaining or some other safer method.